label_sync/repo_labels.yaml

```diff
@@ -0,0 +1,3527 @@
+Labels:
+  apimachinery:
```

Comment:
This file is machine generated and only needed for local testing without consuming GitHub API points.